fails with llvm 17 (and was ignored with 16), actual
__builtin_ms_va_start use is allowed with __attribute__((ms_abi))
--
--- a/configure.ac
+++ b/configure.ac
@@ -1972,7 +1972,6 @@
                    [ac_cv_mabi_ms=yes],[ac_cv_mabi_ms=no])
                 CFLAGS=$saved_CFLAGS])
            AS_VAR_IF([ac_cv_mabi_ms],[yes],
-                     [MSVCRTFLAGS="$MSVCRTFLAGS -mabi=ms"],
                      [test -n "$PE_ARCHS" || AC_MSG_ERROR([The compiler doesn't support -mabi=ms. Use gcc instead of clang, or install mingw-w64.])
                       DLLEXT=""]) ;;
       esac ;;
